### PR TITLE
feat: add log-benchmark stuff for blog

### PR DESCRIPTION
**Pull request by shuiyisong**

### CODE REVIEW

File: log-benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Log benchmark comparison for GreptimeDB
+This repo holds the configuration we used to benchmark GreptimeDB, Clickhouse and Elastic Search.
```
Comment (shuiyisong) [inline log-benchmark/README.md:2]: TODO: add link for en and ch blogs

File: log-benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Steps to reproduce
+0. Decide whether to run structured model test or unstructured mode test.
+1. Build vector binary(see vector's config file for specific branch) and databases binaries accordingly. 
+2. Create table in GreptimeDB and Clickhouse in advance.
+3. Run vector to insert data.
+4. When data insertion is finished, run queries against each database. Note: you'll need to update timerange value after data insertion.
+
+## Addition
+- You can tune GreptimeDB's configuration to get better performance.
```
Comment (shuiyisong) [inline log-benchmark/README.md:31]: TODO: add link for performance tuning blog

File: log-benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+| Clickhouse    | 24.9.1.219 |
+| Elasticsearch | 8.15.0     |
+
+## Creating tables
+See [here](./create_table.sql) for GreptimeDB and Clickhouse's create table clause. 
+The mapping of Elastic search is created automatically.
+
+## Vector Configuration
+We use vector to generate random log data and send inserts to databases.
+Please refer to [structured config](./structured_vector.toml) and [unstructured config](./unstructured_vector.toml) for detailed configuration.
```
Comment (sunng87) [inline log-benchmark/README.md:18]: We need to explain `structured` and `unstructured` first.

**Comment by shuiyisong** [comment]
duplicate of https://github.com/GreptimeTeam/greptimedb/pull/4597

**Pull request closed without merging**